### DISCUSSION TITLE
openkruise 官网无法访问

### DISCUSSION THREAD

**Discussion by skyshaokai** [openkruise/kruise#975] [Q&A]
https://openkruise.io/zh/docs/

<img width="957" alt="image" src="https://user-images.githubusercontent.com/11512967/168561795-4c874009-5a84-466a-8459-7a69bd5e9eb5.png">

**Comment by hantmac** [comment]
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/7600925/168576431-56d580f8-a98d-4439-aad8-9ba003b14eed.png">

可以访问呀
